tensorflow/lite/c/common.h

```diff
@@ -225,6 +225,16 @@ void TfLiteFloatArrayFree(TfLiteFloatArray* a);
     }                                                                      \
   } while (0)
 
+#define TF_LITE_ENSURE_NEAR(context, a, b, epsilon)                    \
+  do {                                                                 \
+    auto delta = ((a) > (b)) ? ((a) - (b)) : ((b) - (a));              \
+    if (delta > epsilon) {                                             \
+      TF_LITE_KERNEL_LOG((context), "%s:%d %s not near %s (%d != %d)", \
+                         __FILE__, __LINE__, #a, #b, (a), (b));        \
```

Comment:
(a) -> static_cast<double>(a)
(b) -> static_cast<double>(b)

once you change the specifier to be %f, it will be expected to be a double. adding a static_cast in the macro definition will allow a caller to pass in floats without an implicit conversion to double error.

Reply:
Done